frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -2134,19 +2159,6 @@ function ReaderFooter:onHoldFooter()
     end
 end
 
-function ReaderFooter:setVisible(visible)
-    if visible then
-        -- If it was off, just do as if we tap'ed on it (so we don't
-        -- duplicate onTapFooter() code)
-        if self.mode == self.mode_list.off then
-            self:onTapFooter(true) -- ignore tap lock
-        end
-        self.view.footer_visible = (self.mode ~= self.mode_list.off)
-    else
-        self:applyFooterMode(self.mode_list.off)
-    end
-end
-
```

Comment:
For people following at home, that's related to the removal of the "hide the footer when the alt bar is enabled" code, which went bye bye in a previous PR ;).